src/BasketBundle/Controller/BasketController.php

```diff
@@ -20,7 +20,7 @@
 use Sonata\Component\Basket\BasketFactoryInterface;
 use Sonata\Component\Customer\AddressInterface;
 use Sonata\Component\Delivery\UndeliverableCountryException;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController as Controller;
```

Comment:
Do not change it. It work diffrence. Add it to master with BlockService update, please.